clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheStateServiceImpl.java

```diff
@@ -242,6 +247,105 @@ private ResourcePageSource createPageSource(String poolName, ServerSideConfigura
+
+    ServerStoreImpl store = stores.get(cacheAlias);
+
+    registerStatistic(pageSource, cacheAlias, "allocatedMemory" ,new Callable<Number>() {
+      @Override
+      public Number call() throws Exception {
+        return store.allocatedMemory();
+      }
+    });
+
+    registerStatistic(pageSource, cacheAlias, "dataOccupiedMemory" ,new Callable<Number>() {
+      @Override
+      public Number call() throws Exception {
+        return store.dataAllocatedMemory();
+      }
+    });
+
+    registerStatistic(pageSource, cacheAlias, "occupiedMemory" ,new Callable<Number>() {
+      @Override
+      public Number call() throws Exception {
+        return store.occupiedMemory();
+      }
+    });
+
+    registerStatistic(pageSource, cacheAlias, "dataOccupiedMemory" ,new Callable<Number>() {
+      @Override
+      public Number call() throws Exception {
+        return store.dataOccupiedMemory();
+      }
+    });
+
+    registerStatistic(pageSource, cacheAlias, "mappings" ,new Callable<Number>() {
+      @Override
+      public Number call() throws Exception {
+        return store.longSize();
+      }
+    });
+
+    registerStatistic(pageSource, cacheAlias, "usedSlotCount" ,new Callable<Number>() {
+      @Override
+      public Number call() throws Exception {
+        return store.usedSlotCount();
+      }
+    });
+
+    registerStatistic(pageSource, cacheAlias, "dataVitalMemory" ,new Callable<Number>() {
+      @Override
+      public Number call() throws Exception {
+        return store.dataVitalMemory();
+      }
+    });
+
+    registerStatistic(pageSource, cacheAlias, "vitalMemory" ,new Callable<Number>() {
+      @Override
+      public Number call() throws Exception {
+        return store.vitalMemory();
+      }
+    });
+
+    registerStatistic(pageSource, cacheAlias, "reprobeLength" ,new Callable<Number>() {
+      @Override
+      public Number call() throws Exception {
+        return store.reprobeLength();
+      }
+    });
+
+    registerStatistic(pageSource, cacheAlias, "removedSlotCount" ,new Callable<Number>() {
+      @Override
+      public Number call() throws Exception {
+        return store.removedSlotCount();
+      }
+    });
+
+    registerStatistic(pageSource, cacheAlias, "dataSize" ,new Callable<Number>() {
+      @Override
+      public Number call() throws Exception {
+        return store.dataSize();
+      }
+    });
+
+    registerStatistic(pageSource, cacheAlias, "tableCapacity" ,new Callable<Number>() {
+      @Override
+      public Number call() throws Exception {
+        return store.tableCapacity();
+      }
+    });
+
+  }
+
+  private void registerStatistic(Object context, String alias, String observerName, Callable callable) {
+    Set<String> tags = new HashSet<String>(Arrays.asList(STATISTICS_TAG,"tier"));
+    Map<String, Object> properties = new HashMap<String, Object>();
+    properties.put("discriminator", STATISTICS_TAG);
+    properties.put("alias", alias);
```

Comment:
change to 
`properties.put("storeName", alias);`
for stores, and
`properties.put("poolName", alias);`
for pools